finetuner/labeler/__init__.py

```diff
@@ -39,6 +39,10 @@ def fit(
     dam_path = tempfile.mkdtemp()
     stop_event = threading.Event()
 
+    # Remove all labels, as they are not needed
+    for doc in train_data:
```

Comment:
Then the argument `clear_labels_on_start` is not used?

Reply:
Oh, haven't seen that one - will fix, thanks